recipes/libinterpolate/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "libinterpolate"
+    description = "A C++ interpolation library with a simple interface that supports multiple interpolation methods."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/CD3/libInterpolate"
+    topics = ("math", "spline", "interpolation", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "19.0",
+            "gcc": "7",
+            "clang": "4",
+            "apple-clang": "10",
+        }
+
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
```

Comment:
Is there a reason you went for `basic_layout` here rather than `cmake_layout`?

I've just noticed that upstream is a cmake library so it might be more appropriate? I've normally seen basic layout used when the upstream project uses autotools or a custom build system

Reply:
Yes, my first contribution was a recipe for cd3-boost-unit-definitions and one of the review comments was that the recipe should use basic_layout because it was a header-only library. I started with the recipe that got accepted for this one.

Honestly, I don't use the layout feature myself (yet), so I don't know which is more appropriate.

Reply:
It's basic layout + turn off a few items in the package info for header-only 

Cmake layout is for cmake build helper :)